src/rp2_common/pico_lwip/lwip.BUILD

```diff
@@ -151,13 +151,13 @@ cc_library(
 
 cc_library(
     name = "pico_lwip_contrib_freertos",
-    srcs = ["ports/freertos/sys_arch.c"],
-    includes = ["ports/freertos/include"],
+    srcs = ["contrib/ports/freertos/sys_arch.c"],
+    includes = ["contrib/ports/freertos/include"],
     target_compatible_with = incompatible_with_config(
```

Comment:
May also need `hdrs = ["contrib/ports/freertos/include/arch/sys_arch.h"],`

Reply:
That header is probably getting picked up by the glob pattern in `pico_lwip_core`, so it might work without this line.

Reply:
It wasn't working with or without, so I reverted this change for some more investigation.